### ISSUE TITLE
🐛 List of unused files is not sorted by path

### ISSUE THREAD

**Issue by kachkaev** [webpro-nl/knip#817]
### Prerequisites

- [X] I'm using the latest version
- [X] I've read the relevant [documentation](https://knip.dev)
- [X] I've searched for [existing issues](https://github.com/webpro-nl/knip/issues?q=is%3Aissue)
- [X] I've checked the [list of known issues](https://knip.dev/reference/known-issues)
- [X] I've read the [issue reproduction guide](https://knip.dev/guides/issue-reproduction)

### Reproduction url

https://codesandbox.io/p/devbox/mrk95r?migrateFrom=7cy572

### Reproduction access

- [X] I've made sure the reproduction is publicly accessible

### Description of the issue

👋 @webpro, glad to see the project rolling! I just configured knip in a new monorepo and noticed that the list of unused files was not sorted by path. As you can see from the reproduction,

```
a/b/d.ts
a/y.ts
a/z.ts
b/a/42.ts
b/x.ts
b/y.ts
c.ts
```

gets printed by `pnpm knip` as:

```
c.ts
a/y.ts
a/z.ts
b/x.ts
b/y.ts
a/b/d.ts 👈  👀 
b/a/42.ts
```

It’d be great if the list was sorted in some consistent way. In the project I am trying to lint I get output like this:

```
apps/foo/src/modules/...
apps/foo/src/modules/...
apps/foo/src/components/...
apps/foo/src/components/...
apps/foo/src/modules/...
apps/foo/src/components/...
apps/foo/src/components/...
...
packages/p4/src/helpers.ts
packages/p1/src/helpers.ts
packages/p3/src/helpers.ts
packages/p2/src/helpers.ts
...
```

Unsorted workspace names and intermixed modules/components are not the end of the world. But it would help if sibling files were shown next to each other so that it was a bit easier to review them.

Hope that it won’t be too hard to fix this problem because unused files are printed synchronously at the end of the analysis rather than on the fly.

Thanks for knip, it’s very useful!

**Comment by webpro** [comment]
Kinda surprised we didn't do this yet :) Starting out with a simple sort tho:

```
a.js
b/d.ts
c.js
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.34.3. See [Release 5.34.3](https://github.com/webpro-nl/knip/releases/tag/5.34.3) for release notes._

_Using Knip in a commercial project? Please consider [becoming a sponsor](https://knip.dev/sponsors)._
